Marlin/Marlin.h

```diff
@@ -293,6 +293,15 @@ extern float min_pos[3];
 extern float max_pos[3];
 extern bool axis_known_position[3];
 
+//this is for storing the position on a card so that when turned off the position can be restored without using the z endstop.
+#ifdef SDSUPPORT
```

Comment:
You should write this as an optional feature by adding a new option in `Configuration_adv.h` in the `SDSUPPORT` block. For example:
```
#ifdef SDSUPPORT
  . . .
  //#define SD_SAVE_AXIS_POSITIONS
  . . .
#endif
```